docs/src/man/subsets.md

```diff
@@ -6,7 +6,7 @@ The `DataArray` type is meant to behave like a standard Julia `Array` and tries
 
 One dimensional `DataArray`:
 
-```julia
```

Comment:
Yes, they should work fine now (I pushed a fix for doctests yesterday), so long as you match the expected output exactly. I'll see if I can get some of them working locally.

Code highlighting probably won't work without the `julia`.